core.spec.md

```diff
@@ -0,0 +1,501 @@
+<code class=anatomy>
+  <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>identity</aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#frag<aside>ignored</aside></span>
+</code>
+```
+
+#### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the feature used by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the {@core} directive would prevent this.
+
+###! as: String
+
+Change the [names](#sec-Prefixing) of directives and schema elements from this specification. The specified string MUST be a valid GraphQL identifier and MUST NOT contain the namespace separator (two underscores, {"__"}).
+
+When [`as:`](#@core/as) is provided, processors MUST replace the default name prefix on the names of all [prefixed schema elements](#sec-Elements-which-must-be-prefixed) with the specified name.
+
+```graphql example -- Using {@core}(feature:, as:) to use a feature with a custom name
+schema
+  @core(feature: "https://spec.example.com/core/v1.0")
+  @core(feature: "https://spec.example.com/example/v1.0", as: "eg")
+{
+  query: Query
+}
+
+type User {
+  # Specifying `as: "eg"` transforms @example into @eg
+  name: String @eg(data: ITEM)
+}
+
+# Additional specified schema elements must have their prefixes set
+# to the new name.
+#
+# This data enum was specified as `example__Data`, but will be renamed
+# as `eg__Data`:
+enum eg__Data {
+  ITEM
+}
+
+# Name transformation must also be applied to definitions pulled in from
+# specifications.
+directive @eg(data: eg__Data) on FIELD
+
+# (...other definitions omitted...)
+```
+
+###! export: Boolean! = false
+
+Control whether the entire feature is exported to the API.
+
+If {true}, all schema elements--including all directive usages--from this feature will be included in the schema's API *unless* they are annotated with {@core__export}`(export: false)`.
+
+If {false} (the default), no schema elements from this feature will be included in the schema's API *unless* they are annotated with {@core__export}.
+
+##! @core__export
+
+Specify whether the annotated element is exported to the API.
+
+```graphql definition
+directive @core__export(export: Boolean! = true)
+  repeatable on
+  | SCALAR
+  | OBJECT
+  | FIELD_DEFINITION
+  | ARGUMENT_DEFINITION
+  | INTERFACE
+  | UNION
+  | ENUM
+  | ENUM_VALUE
+  | INPUT_OBJECT
+  | INPUT_FIELD_DEFINITION
+```
+
+{@core__export} can occur at any type system location. Elements with {@core__export} will always be included in the API. Elements with {@core__export}`(export: false)` will always be excluded from the API.
```

Comment:
SUBSTANTIVE: Probably worth noting that your declaration does not need to list locations that you don't use in your document.